### DISCUSSION TITLE
Can we get a release with the most recent PRs?

### DISCUSSION THREAD

**Discussion by lawik** [dsheiko/puppetry#100] [General]
Hey @dsheiko 

As you might remember I contributed #91 and #92. Will those be in a release anytime soon? We would really benefit from having them in our work using Puppetry.

**Comment by dsheiko** [comment]
Sorry I've busy recently. But I started already to combine fixes for the next release.

**Reply by lawik** [reply]
Great!

**Comment by lawik** [comment]
Hey, saw that you released v3.2.6, that's great. Includes some good stuff.

Any reason #91 and #92 aren't in it?

**Comment by dsheiko** [comment]
Both changes are actually present within the release (I probably messed up with git a bit, but nonetheless they are in the code)

For #91 please check
https://github.com/dsheiko/puppetry/blob/master/src/service/assert.js#L78
that is  exactly the change you did

#92 is also in place, albeit with changes
https://github.com/dsheiko/puppetry/blob/master/src/component/Schema/Params/Element/upload.js#L12
it now considers string a relative path if it starts with ./ otherwise takes it as it is. Older way it tried to find the path, what blocked from using paths specific for environment.

**Reply by lawik** [reply]
Oh, great. Yeah I just checked git commits.
